Integrations/integration-Wildfire.yml

```diff
@@ -136,14 +146,47 @@ script:
         var prettyUploadBody = {
             'MD5': body.md5,
             'SHA256': body.sha256,
-            'FileType': body.filetype,
-            'Size': body.size,
             'Status': 'Pending'
         };
+        if (body.filetype) {
+            prettyUploadBody.FileType = body.filetype;
+        }
+        if (body.size) {
+            prettyUploadBody.Size = body.size;
+        }
+        if (body.url) {
+            prettyUploadBody.URL = body.url;
+        }
         return prettyUploadBody;
     }
 
 
+    function prettifyVerdict(verdictData) {
+        var prettyVerdict = {};
+        if (verdictData.md5) {
+            prettyVerdict.MD5 = verdictData.md5;
+        }
+        if (verdictData.sha256) {
+            prettyVerdict.SHA256 = verdictData.sha256;
+        }
+        prettyVerdict.Verdict = Number(verdictData.verdict);
+        prettyVerdict.VerdictDescription = VERDICTS_DICT[verdictData.verdict];
+
+        return prettyVerdict;
+    }
+
+
+    function createDBotScoreFromVerdict(prettyVerdict) {
+        var DbotScore = {
+            'Indicator': prettyVerdict.SHA256? prettyVerdict.SHA256 : prettyVerdict.MD5,
```

Comment:
This function needs to log error in case hash is missing or in case VERDICTS_TO_DBOTSCORE will not have a value for the key.

Reply:
will add this check here here and also in `createDBotScoreFromVerdicts`
product writer will phrase the error